frontend/scripts/nlp-algorithm/NLP.py

```diff
@@ -0,0 +1,2329 @@
 need to keep the exponents looking exactly like they were given to you in the original problem.", "#3 is incorrect, 60 is a high number to multiply with, try 4?", "#1 looking for time (so h)", "# 10 is incorrect", "#11 - x=2 is correct BUT you then need to plug it in for FG = 11(2)-7=15 (-1)", "#24) dit des mots doux", "#5 since velocity was constant (no acceleration) between 2 and 14 s, it would have a horizontal line at 1.0 m/s", "#2 correct setup, incorrect final answer -1", "#16 - These are not angles. (-1)", "$$(2)(\u00bd) = 1,$$. not. $$2.5$$.", "$$A=36piy^8$$ units^2", "#5 is incorrect and you need to put all answers in correct form.  Please revise and resubmit for grading.  Thanks.", "#4  should you X ?  (-7)", "#6: Excellent ability to identify functional groups and to deduce solubility.", "#8 (-8)", "$$\\kappa$$ is scalar", "$$3(49x^6)$$", "#2  also a, c, d  (-6)", "#4e: One difference is that salicylic acid contains an alcohol group so it contains a strong, broad peak in the 3200-3600 cm-1 range.", "#13 x=15 is correct here as well. Great Job!", "\"their entering son/son who is entering\"", "#3) The layer with the highest oxygen, and lowest sulfide is layer 1. The layer with the lowest oxygen and highest sulfide is layer 7.", "#3 see rounding note above -1", "$$-5^3$$ not $$-5*3$$", "#23 (-1)", "#2 acute Isos (-4)", "#8)  Line 4.  When we have that triangles are congruent we will always be using one of our shortcuts that proves triangles congruent.", "#2 9/9", "#4c incorrect", "$$  a = (Vf - Vi)/t $$", "#4. Calculation exact. Estimate, not. Reasonable estimate would be 45 divided by 9.", "#7b:  Round to 714 people", "#1:  Plug into your calculator:  $$log16/log5$$.  Since there is not base given, we assume the base to be 10.  However, your calculator will let the base be 10 on its own.", "#7:  The answer to this problem is (5,6).", "#1)  Your height is not correct.  You need to find the side adjacent to the 60\u00ba angle first then use trig.", "#3 7/9", "\"...chase\" (Orwell 52-53).", "\"da paseos\" or \"monta/anda\"", "#8 lots of hard work, try to get to 16, 500 cu ft (-8)", "\"Shouldn't\" doesnt work here - its I \"wouldn't\" worry", "#11: Make sure to write your answer as a statement", "$$-y=4$$", "#7)  Watch your notation here you should have a ray", "\"and so' is not proper.... Keep in mind ...we don't write they way we speak", "#4 (-4)", "#5&6: The endpoints and acid-base indicator labels are missing, (-0.5 for each missing label).", "#6-3: Remember to multiply the answer from part 2 by five due to the dilution prior to dividing by 0.83 mol/L", "$$28^2$$ is not $$28 times 2$$", "\"az\" nem kell", "#4 is not correct", "$$(x-7)(x+4)$$", "#1 set up like algebra equation, then solve...-10", "#5 0/9", "\"open\" only", "#2c: The entire substance; Cu2CO3(OH)2(s) has been reduced, not just Cu(s). 0.5/1", "#2:  The setup looks good.  You need to calculate the solution to this.", "#1 (-1) Dont forget the square root symbol", "#4   Of alw
```

Comment:
Separate the data from the code. This is not professional :)

Reply:
Sure, adding these files actually was out of the scope of this sprint, however, I add them for future reference. Thanks.